### PR TITLE
WIP: Translate go CLI

### PR DESCRIPTION
**Pull request by Jorropo**

### CODE REVIEW

File: go/internal/initutil/ipfs.go [hunk 1]
```diff
@@ -32,9 +32,9 @@ import (
 )
 
 func (m *Manager) SetupLocalIPFSFlags(fs *flag.FlagSet) {
-	fs.StringVar(&m.Node.Protocol.IPFSListeners, "p2p.ipfs-listeners", "/ip4/0.0.0.0/tcp/0,/ip4/0.0.0.0/udp/0/quic", "IPFS listeners")
-	fs.StringVar(&m.Node.Protocol.IPFSAPIListeners, "p2p.ipfs-api-listeners", "", "IPFS API listeners")
-	fs.StringVar(&m.Node.Protocol.Announce, "p2p.ipfs-announce", "", "IPFS announce addrs")
```
Comment (moul) [inline go/internal/initutil/ipfs.go:37]: the API is ugly to use in go, we need something like `t("cli.ipfs.listeners")`

File: go/internal/i18n/locate_lang.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package main
```
Comment (moul) [inline go/internal/i18n/locate_lang.go:1]: no `main` package in `go/internal`

File: go/internal/i18n/generate-json.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
```
Comment (moul) [inline go/internal/i18n/generate-json.py:1]: avoid python if possible, and especially in the `go/` dir

File: .github/workflows/go-i18n.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+name: Go i18n
```
Comment (moul) [inline .github/workflows/go-i18n.yml:1]: avoid creating new workflows, write new jobs in existing ones instead

**Review by moul — CHANGES_REQUESTED** [review]
see review comments

btw, I'm for closing this since it's ultra low-priority and adding useless complexity during development

File: go/internal/i18n/locate_lang.go (hunk 2 above)
Comment (Jorropo) [reply go/internal/i18n/locate_lang.go:1]: Yes sorry, I was debbuging (it's not ready yet).

File: go/internal/i18n/generate-json.py (hunk 3 above)
Comment (Jorropo) [reply go/internal/i18n/generate-json.py:1]: Can I move it in a folder under `/tool/` ?

File: .github/workflows/go-i18n.yml (hunk 4 above)
Comment (Jorropo) [reply .github/workflows/go-i18n.yml:1]: That was for having a custom path trigger, this workflows need to be runned only if some touches `*/i18n.go`.

**Pull request closed without merging**